framework/src/modules/dpos_v2/schemas.ts

```diff
@@ -12,36 +12,35 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
-import { MAX_LENGTH_NAME } from './constants';
+import { BLS_PUBLIC_KEY_LENGTH, BLS_POP_LENGTH, ED25519_PUBLIC_KEY_LENGTH } from './constants';
 
 export const delegateRegistrationCommandParamsSchema = {
 	$id: '/dpos/command/registerDelegateParams',
 	type: 'object',
-	required: ['name', 'generatorKey', 'blsKey', 'proofOfPossession'],
+	required: ['name', 'blsKey', 'proofOfPossession', 'generatorKey', 'delegateRegistrationFee'],
 	properties: {
 		name: {
 			dataType: 'string',
 			fieldNumber: 1,
-			minLength: 1,
-			maxLength: MAX_LENGTH_NAME,
 		},
-		generatorKey: {
+		blsKey: {
 			dataType: 'bytes',
+			length: BLS_PUBLIC_KEY_LENGTH,
```

Comment:
`length` is not valid JSON schema. Please keep `minLength`, `maxLength`

Reply:
I copied that from the LIP:

https://github.com/LiskHQ/lips-staging/blob/main/proposals/lip-0057.md#parameters

The LIP is then incorrect and should be updated?

Reply:
LIP is just saying the length should be as defined. How it's validated is up to the implementation.
the implementation wise, it needs to use min/max because the Ajv we use don't have `length` property